packages/sdk/src/wallet-utils/encode-provable-authn.js

```diff
@@ -0,0 +1,10 @@
+import {withPrefix} from "@onflow/util-address"
+import {invariant} from "@onflow/util-invariant"
+import {encode} from '@onflow/rlp';
+
+export const encodeMessageFromProvableAuthn = (address, timestamp, appTag = "") => {
```

Comment:
That's a mouthful!
Can you export from `wallet-utils/index` for use as WalletUtils.encodeMessageFromProvableAuthn()

Reply:
Yea the name is a bit lengthy... 🤔 i'm open to suggestions!